sandbox/src/App.js

```diff
@@ -59,6 +61,9 @@ function BasicExample() {
           <li>
             <a href="/legacy.html">Legacy Visitor ID</a>
           </li>
+          <li>
+            <a href="/redirectOffers">Redirect Offers</a>
+          </li>
```

Comment:
You'll probably need to update the menu found within legacy.html as well. Keeping the menu in sync in these two files is a chore, but hopefully will be improved in https://jira.corp.adobe.com/browse/PDCL-4822.

Reply:
thank you